vivisect/__init__.py

```diff
@@ -714,7 +717,7 @@ def addExport(self, va, etype, name, filename):
         """
         Add an already created export object.
         """
-        rname = "%s.%s" % (filename,name)
+        rname = "%s.%s_%x" % (filename,name,va)
```

Comment:
let me know if this is a showstopper.  i keep running into the same name at different locs.  i want both.  i can add this to another PR if you prefer.

Reply:
Seems like it's no longer valid? Or you moved it to another PR.

I'm fine with this being another PR.